ownCloud/Settings/Passcode/PasscodeViewController.swift

```diff
@@ -257,4 +265,54 @@ class PasscodeViewController: UIViewController, Themeable {
 		cancelButton?.applyThemeCollection(collection, itemStyle: .defaultForItem)
 		cancelButton?.layer.cornerRadius = 0
 	}
+    
+    // MARK: - External Keyboard Commands
+    
+    @objc func performKeyCommand(sender: UIKeyCommand) {
+        guard let key = sender.input else {
+            return
+        }
+        
+        switch key {
+        case "\u{8}":
```

Comment:
Maybe you could code `"\u{8}"` into a constant to provide more information. Then you can reuse this constant in line `300`

Reply:
Thank you for pointing to this improvement